### PR TITLE
fix: board filters

### PR DESCRIPTION
**Pull request by raylu**

### CODE REVIEW

File: internal/jira/jira_boards.go [hunk 1]
```diff
@@ -39,3 +40,16 @@ func (api *httpApi) GetBoardConfiguration(boardId int) (*BoardConfiguration, err
 	}
 	return &result, nil
 }
+
+func (api *httpApi) GetFilter(filterId string) (*Filter, error) {
```
Comment (mk-5) [inline internal/jira/jira_boards.go:44]: the workflow goes as follow:

-> select project
-> show boards for that project

I don't see how to work with that `GetFilter(filterId string)` method. How to get appropriate filter id upfront?

Comment (mk-5) [reply internal/jira/jira_boards.go:44]: ah okay I see it now - it's in goto.go

File: internal/boards/goto.go [hunk 2]
```diff
@@ -23,8 +23,9 @@ func RegisterGoTo() {
 			app.Error(err.Error())
 			return
 		}
+		filter, err := api.GetFilter(boardConfig.Filter.Id)
```
Comment (mk-5) [inline internal/boards/goto.go:26]: err is ignored here

Comment (raylu) [reply internal/boards/goto.go:26]: good catch! 8a3cc4023cc74ecef7c691bc8b2042fee97fd61b

**Comment by mk-5** [comment]
@raylu okay - I think it's ready to be merged, but github actions are still missing. I think it's because all actions are set to `on: push` instead of `on: pull_request`. I will try to change it, and I'll ask you to rebase again 🤞

**Comment by mk-5** [comment]
@raylu

okay there is one failure about commit message. The second one with Goreleaser - Check / it's something I need to fix - but I can ignore that check for now. There is a problem with some of the secrets that are loaded for that workflow. I don't know why (yet), but it doesn't work when PR is coming from forked repos.

File: internal/jira/jira_boards.go (hunk 1 above)
Comment (mk-5) [inline internal/jira/jira_boards.go:44]: would be nice also to add some test for that method  - I'll merge it, and I'll add some test later

**Comment by mk-5** [comment]
there is some regression after that feature. I see tickets from backlog as a first column. I will try to fix it

**Pull request merged**